client/nginx.go

```diff
@@ -12,7 +13,7 @@ import (
 )
 
 const (
-	// APIVersion is a version of NGINX Plus API.
+	// APIVersion is a current version of NGINX Plus API.
 	APIVersion = 5
```

Comment:
Optional suggestion:
```suggestion
	DefaultAPIVersion = 5
```

Reply:
I think that's more descriptive @Dean-Coakley, looks like a good change

Reply:
I think @framer777 had some concerns:
 
Considering that APIVersion is an exported variable, there could be dependencies on the variable in other projects which this change would break/require fixing.

As an alternative he updated the doc string:
`// APIVersion is a version of NGINX Plus API.` -> `// APIVersion is the default version of NGINX Plus API supported by the client.`

Reply:
Ah a very good point about it being exported, makes sense!